api/v1alpha1/istiotypes/istio_settings.go

```diff
@@ -0,0 +1,33 @@
+package istiotypes
+
+// This package reimplements specific parts of the Telemetry struct from "istio.io/api/telemetry/v1" in order to be
+// forward compatible if we choose to incorporate the struct directly in the future.
+
+// Tracing contains relevant settings for tracing in the telemetry configuration
+// NB: RandomSamplingPercentage actually uses a wrapped type of *wrappers.DoubleValue in the original struct, but due
+// to incompatibalities with the kubebuilder code generator, we have chosen to use a simple int instead. This only allows
+// for whole numbers, but this is sufficient for our use case.
+//
+// +kubebuilder:object:generate=true
+type Tracing struct {
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=100
+	// +kubebuilder:default:=10
+	RandomSamplingPercentage int `json:"randomSamplingPercentage,omitempty"`
```

Comment:
// +kubebuilder:validation:Maximum=1000
// +kubebuilder:default:=100
```suggestion
	RandomSamplingPerMille int `json:"randomSamplingPercentage,omitempty"`
```

Reply:
just a thought. might be confusing tho

Reply:
> Not a bad idea if we want to support decimal values, but that would cause an api-change if we later use the istio-types directly later because of the name change. Do you think we would need sub 1% sampling?

No idea :smile: